estate/models/estate_property.py

```diff
@@ -0,0 +1,45 @@
+from odoo import fields, models
+
+
+class estate_property(models.Model):
+
+    _name = "estate.property"
+    _description = "The estate property"
+    name = fields.Char("Property name", required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    active = fields.Boolean(default=True)
+    date_availability = fields.Date(
+        default=fields.Date.add(fields.Date.today(), months=3)
+    )
```

Comment:
May be easier to read if oneline. Usually, we tend to say max 100 characters per line, then line break
```suggestion
    date_availability = fields.Date( default=fields.Date.add(fields.Date.today(), months=3))
```